frontend/src/component/personalDashboard/ConnectSDK.tsx

```diff
@@ -35,10 +35,10 @@ export const CreateFlag: FC<{ project: string }> = ({ project }) => {
                 <NeutralCircleContainer>1</NeutralCircleContainer>
                 Create a feature flag
             </TitleContainer>
-            <Typography>
+            <div>
                 <div>The project currently holds no feature toggles.</div>
                 <div>Create a feature flag to get started.</div>
-            </Typography>
+            </div>
```

Comment:
I think you might have this backward? Why not use a wrapping div and two `p` children (or typography, as it were)?